projects/plugins/jetpack/modules/wpcom-block-editor/functions.editor-type.php

```diff
@@ -3,11 +3,12 @@
  * This file contains some 'remember' functions inspired by the core Classic Editor Plugin
  * Used to align the 'last editor' metadata so that it is set on all Jetpack and WPCOM sites
  *
+ * @deprecated $$next-version$$
  * @package automattic/jetpack
  */
-
 namespace Jetpack\EditorType;
 
+_deprecated_file( __FILE__, 'jetpack-$$next-version$$', 'Automattic\\Jetpack\\Jetpack_Mu_Wpcom\\WPCOM_Block_Editor\\EditorType' );
```

Comment:
I have marked this file as deprecated too. This implies that we need to take care of only existing use I have been able to find since now it is using deprecated code. D149564-code should take care of it.